.github/workflows/e2e-tests.yml

```diff
@@ -144,6 +144,10 @@ jobs:
       env:
         TERM: xterm
 
+    - name: Percy Test Sanity Check
+      if: matrix.edition == 'ee'
```

Comment:
We don't have any visual test that is OSS-specific so this "optimization" should be enough for now.